instrumentation/lettuce/lettuce-5.0/src/test/groovy/LettuceSyncClientTest.groovy

```diff
@@ -30,320 +31,320 @@ import java.util.concurrent.CompletionException
 import static io.opentelemetry.trace.Span.Kind.CLIENT
 
 class LettuceSyncClientTest extends AgentTestRunner {
-  public static final String PEER_NAME = "localhost"
-  public static final String PEER_IP = "127.0.0.1"
-  public static final int DB_INDEX = 0
-  // Disable autoreconnect so we do not get stray traces popping up on server shutdown
-  public static final ClientOptions CLIENT_OPTIONS = ClientOptions.builder().autoReconnect(false).build()
-
-  @Shared
-  int port
-  @Shared
-  int incorrectPort
-  @Shared
-  String dbAddr
-  @Shared
-  String dbAddrNonExistent
-  @Shared
-  String dbUriNonExistent
-  @Shared
-  String embeddedDbUri
-
-  @Shared
-  RedisServer redisServer
-
-  @Shared
-  Map<String, String> testHashMap = [
-    firstname: "John",
-    lastname : "Doe",
-    age      : "53"
-  ]
-
-  RedisClient redisClient
-  StatefulConnection connection
-  RedisCommands<String, ?> syncCommands
-
-  def setupSpec() {
-    port = PortUtils.randomOpenPort()
-    incorrectPort = PortUtils.randomOpenPort()
-    dbAddr = PEER_NAME + ":" + port + "/" + DB_INDEX
-    dbAddrNonExistent = PEER_NAME + ":" + incorrectPort + "/" + DB_INDEX
-    dbUriNonExistent = "redis://" + dbAddrNonExistent
-    embeddedDbUri = "redis://" + dbAddr
-
-    redisServer = RedisServer.builder()
-    // bind to localhost to avoid firewall popup
-      .setting("bind " + PEER_NAME)
-    // set max memory to avoid problems in CI
-      .setting("maxmemory 128M")
-      .port(port).build()
-  }
-
-  def setup() {
-    redisClient = RedisClient.create(embeddedDbUri)
-
-    redisServer.start()
-    connection = redisClient.connect()
-    syncCommands = connection.sync()
-
-    syncCommands.set("TESTKEY", "TESTVAL")
-    syncCommands.hmset("TESTHM", testHashMap)
-
-    // 2 sets + 1 connect trace
-    TEST_WRITER.waitForTraces(3)
-    TEST_WRITER.clear()
-  }
-
-  def cleanup() {
-    connection.close()
-    redisServer.stop()
-  }
-
-  def "connect"() {
-    setup:
-    RedisClient testConnectionClient = RedisClient.create(embeddedDbUri)
-    testConnectionClient.setOptions(CLIENT_OPTIONS)
-
-    when:
-    StatefulConnection connection = testConnectionClient.connect()
-
-    then:
-    assertTraces(1) {
-      trace(0, 1) {
-        span(0) {
-          operationName "CONNECT"
-          spanKind CLIENT
-          errored false
-          attributes {
-            "${SemanticAttributes.NET_PEER_NAME.key()}" PEER_NAME
-            "${SemanticAttributes.NET_PEER_IP.key()}" PEER_IP
-            "${SemanticAttributes.NET_PEER_PORT.key()}" port
-            "${SemanticAttributes.DB_TYPE.key()}" "redis"
-            "${SemanticAttributes.DB_STATEMENT.key()}" "CONNECT"
-            "db.redis.dbIndex" 0
-          }
+    public static final String PEER_NAME = "localhost"
```

Comment:
can you try reformatting this file in Intellij? i'm not sure why it's not using our `.editorconfig`

Reply:
reformat doesn't seem to change anything.